Makefile

```diff
@@ -21,15 +20,14 @@ show-version: $(GOBIN)/gobump
 	@gobump show -r .
 
 $(GOBIN)/gobump:
-	@cd && go get github.com/x-motemen/gobump/cmd/gobump
+	@go install github.com/x-motemen/gobump/cmd/gobump@latest
 
 .PHONY: cross
 cross: $(GOBIN)/goxz
-	goxz -n $(BIN) -os linux,darwin,windows -arch amd64 -pv=v$(VERSION) -build-ldflags=$(BUILD_LDFLAGS) .
-	goxz -n $(BIN) -os linux,darwin -arch arm64 -pv=v$(VERSION) -build-ldflags=$(BUILD_LDFLAGS) .
+	goxz -n $(BIN) -pv=v$(VERSION) -build-ldflags=$(BUILD_LDFLAGS) .
```

Comment:
See https://github.com/Songmu/goxz/blob/main/CHANGELOG.md#v080-2021-12-26, goxz builds both amd64 and arm64 by default.